packages/alert-dialog/src/AlertDialog.ts

```diff
@@ -20,6 +20,9 @@ import {
     query,
 } from '@spectrum-web-components/base/src/decorators.js';
 import '@spectrum-web-components/button/sp-button.js';
+import '@spectrum-web-components/button-group/sp-button-group.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-alert.js';
+import '@spectrum-web-components/divider/sp-divider.js';
```

Comment:
This fixes the UI issue in AlertDialog that's why you are seeing a new baseline update on VRT.